lib/client/ruby/tunnel-vision.rb

```diff
@@ -26,42 +26,62 @@ class TunnelVision < Thor
 
   class InvalidArgument < ::StandardError; end
 
-  CONFIG_URL = 'https://raw.githubusercontent.com/abhchand/tunnel-vision/master/roles/tunnel-server/vars/main.yml'.freeze
-  TUNNEL_URL = 'exec@pipe.cr-tunnel.xyz'.freeze
+  TUNNEL_HOST = 'pipe.cr-tunnel.xyz'.freeze
+  TUNNEL_USER = 'exec'.freeze
+  VERSION = '0.1.0'
```

Comment:
Started tracking a `version` of the client.

Reply:
good idea 💯